index.html

```diff
@@ -67,7 +67,7 @@
     <link rel="stylesheet" href="css/app.css?v=[[VERSION]]">
 </head>
 
-<body ng-controller="ThemeController as themeCtrl" class="{{ ctrl.statusClass }}" ng-class="{expanded: ctrl.expandStatusBar}" md-theme="{{ themeCtrl.theme }}">
+<body ng-controller="ThemeController as themeCtrl" class="{{ [ctrl.statusClass, themeCtrl.userInterfaceClass].join(' ') }}" ng-class="{expanded: ctrl.expandStatusBar}" md-theme="{{ themeCtrl.theme }}">
```

Comment:
Would `class="{{ ctrl.statusClass }} {{ themeCtrl.userInterfaceClass }}"` work as well? That would be a bit easier to read.